src/course-search/SearchBox.jsx

```diff
@@ -40,7 +40,7 @@ export const SearchBoxBase = ({
   return (
     <div className={className}>
       {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-      <label id="search-input-box" className="fe__searchfield-input-box">
+      <label id="search-input-box" className="fe__searchfield-input-box text-brand-primary">
```

Comment:
This will fix the issue where the "Search courses" label on the search box isn't turning white on darker backgrounds.

Reply:
Thanks, sorry I didn't get to it!

Reply:
No worries!